subsys/bluetooth/controller/ll_sw/ti/hal/cc13xx_cc26xx/radio/RFQueue.c

```diff
@@ -0,0 +1,133 @@
+/******************************************************************************
```

Comment:
This will have to move to external repo due to different license.

Reply:
@vanti see https://github.com/vanti/hal_ti/pull/4